spyder/widgets/helperwidgets.py

```diff
@@ -455,5 +459,88 @@ def test_msgcheckbox():
     box.exec_()
 
 
+class PanelEmptyWidget(QFrame):
+    """Widget to show a panel/plugin functionality description."""
```

Comment:
```suggestion
class PaneEmptyWidget(QFrame):
    """Widget to show a pane/plugin functionality description."""
```

* I suggest to rename this class to `PaneEmptyWidget` to not get things confused with the Editor panels.
* All references in this PR to `panel` should be renamed to `pane` too (I didn't leave suggestions for all of them because there are too many).